flexget/components/notify/notifiers/gotify.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import base64
+import datetime
+import re
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+from flexget.plugin import PluginWarning
+from flexget.utils.requests import Session as RequestSession, TimedLimiter
+from requests.exceptions import RequestException
+
+plugin_name = 'gotify'
+log = logging.getLogger(plugin_name)
+
+requests = RequestSession(max_retries=3)
+
+gotify_url_pattern = {
+    'type': 'string',
+    'pattern': r'^http(s?)\:\/\/.*\/message$',
+    'error_pattern': 'Gotify URL must begin with http(s) and end with `/message`',
+}
+
+class GotifyNotifier(object):
+    """
+    Example::
+
+    notify:
+      entries:
+        via:
+          - gotify:
+              url: <GOTIFY_SERVER_URL>
+              token: <GOTIFY_TOKEN>
+              priority: <PRIORITY>
+    Configuration parameters are also supported from entries (eg. through set).
+    """
+    schema = {
+        'type': 'object',
+        'properties': {
+            'url': gotify_url_pattern,
```

Comment:
i'm thining using a rege here is probably overkill. You can use `'format': 'url'` here and just do a urljoin to add the `message` suffix

Reply:
I think I originally didn't want to try to make assumptions about the URL since it's self-hosted, but I think you're right that it's less confusing to tell users to find their own `/messages` URL (this assumes that people would have knowledge of using the API directly). However having flexget append that is more similar to how people set up the Android app, will do that.

Reply:
https://github.com/Flexget/Flexget/pull/2533/commits/4971f0ee7c3f2b9ab48d3c6e0947d25e9ad5c82d